src/aleph_client/commands/instance/__init__.py

```diff
@@ -212,18 +202,31 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
             if not firmware_message:
                 typer.echo("Confidential Firmware hash does not exist on aleph.im")
                 raise typer.Exit(code=1)
+    if not name:
+        name = validated_prompt("Instance name", lambda x: len(x) < 65)
```

Comment:
What about simplifying this ?
```suggestion
    name = name or validated_prompt("Instance name", lambda x: len(x) < 65)
```